packages/page-explorer/src/BlockHeader.tsx

```diff
@@ -6,17 +6,13 @@ import type {HeaderExtended} from '@polkadot/api-derive/types';
 import React from 'react';
 import {Link} from 'react-router-dom';
 
-import {AddressSmall} from '@polkadot/react-components';
-import {useBlockAuthor} from '@polkadot/react-hooks/useBlockAuthor';
 import {formatNumber} from '@polkadot/util';
 
 interface Props {
   value: HeaderExtended;
 }
 
 function BlockHeader({value}: Props): React.ReactElement<Props> | null {
-  const author = useBlockAuthor(value);
```

Comment:
Please comment out instead of removing, the other relevant section below is commented-out.